Framework/Algorithms/src/CreateUserDefinedBackground.cpp

```diff
@@ -187,6 +180,61 @@ CreateUserDefinedBackground::createBackgroundWorkspace(
 }
 
 /**
+ * Get Y storage mode for background data.
+ *
+ * - Point data -> counts
+ * - Histogram and distribution -> frequencies
+ * - Histogram, not distribution -> counts, BUT if the data was plotted using
+ * the "normalize histogram to bin width" option then it needs to be multiplied
+ * by bin width
+ *
+ * @param data :: [input] Input workspace
+ * @param yBackground :: [input, output] Y background data - could be changed
+ * depending on options
+ * @returns :: Y storage mode to be used
+ */
+HistogramData::Histogram::YMode CreateUserDefinedBackground::getBackgroundYMode(
+    const API::MatrixWorkspace_const_sptr &data,
+    std::vector<double> &yBackground) const {
+  if (data->isHistogramData()) {
+    if (data->isDistribution()) {
+      return Histogram::YMode::Frequencies;
+    } else {
+      if ("On" ==
+          Kernel::ConfigService::Instance().getString(AUTODISTRIBUTIONKEY)) {
+        // User has selected the option "normalize histogram to bin width"
+        multiplyByBinWidth(data, yBackground);
+      }
+      return Histogram::YMode::Counts;
+    }
+  } else {
+    return Histogram::YMode::Counts;
```

Comment:
Is this correct? I thought point-data does often correspond to frequencies, not counts?